src/library/components/page-header.js

```diff
@@ -107,40 +99,31 @@ var PageHeader = Component({
 
   renderFirstButton: function () {
     if (this.state.loggedIn) {
-      return a({href: this.state.homePath, title: 'View Recent Activity', className: 'portal-pages-main-nav-item__link button register'},
-        i({className: 'icon-home'}),
-        'My Classes'
+      return (
+        <a href={this.state.homePath} title={'View Recent Activity'} className={'portal-pages-main-nav-item__link button register'}><i className={'icon-home'}></i>{'My Classes'}</a>
```

Comment:
I think you can replace `{'My Classes'}` with just `My Classes`.  If that works then you can do the same with the other links below.